velox/docs/index.rst

```diff
@@ -9,6 +9,7 @@ Velox Documentation
     monthly-updates
     functions
     spark_functions
+    iceberg_functions
```

Comment:
<img width="275" alt="Screenshot 2025-06-04 at 16 58 12" src="https://github.com/user-attachments/assets/809a3790-d633-46be-b1f9-f5e05ebacea3" />

Do we need the same subtitle `Iceberg Functions`?

Reply:
1 page is enough, iceberg only has 6 functions